src/mixins/atoms.js

```diff
@@ -88,13 +88,50 @@ export const AtomsMixin = (superclass) =>
         createAtomsGroup(basis, atomRadiiScale) {
             const atomsGroup = new THREE.Group();
             atomsGroup.name = ATOM_GROUP_NAME;
+            const { atomicLabelsArray, elementsWithLabelsArray } = basis;
             basis.coordinates.forEach((atomicCoordinate, atomicIndex) => {
                 const element = basis.getElementByIndex(atomicIndex);
                 const sphereMesh = this.getSphereMeshObject({
                     ...this._getDefaultSettingsForElement(element, atomRadiiScale),
                     coordinate: atomicCoordinate.value,
                 });
                 sphereMesh.name = `${element}-${atomicIndex}`;
+                sphereMesh.userData = {
+                    ...sphereMesh.userData,
+                    symbolWithLabel: elementsWithLabelsArray[atomicIndex],
+                };
+                // set glow according to the labels, currently only single digit
+                // numeric labels are allowed, in practice we expect only two
+                // different labels: 1 and 2 for up and down spin representations
+                const atomColor = this.getAtomColorByElement(element).toLowerCase();
```

Comment:
Let's isolate the code that sets glow to a separate function and import from utils maye. Maybe even put to cove.js or another library